src/main/java/edu/hw3/Task1.kt

```diff
@@ -0,0 +1,38 @@
+package edu.hw3
+
+class Task1 {
+    private val lowers: MutableList<Char> = ArrayList()
+    private val uppers: MutableList<Char> = ArrayList()
+    private val lowersStart = 'a'.code
+    private val uppersStart = 'A'.code
+
+    init {
+        for (i in 0..25) {
+            lowers += (lowersStart + i).toChar()
+            uppers += (uppersStart + i).toChar()
+        }
+    }
+
+    fun atbash(s: String): String {
+        val res = StringBuilder()
```

Comment:
`Enhacement`, мб сделаешь через 
```suggestion
        val res = StringBuilder().apply{}
```?

Reply:
нашел метод buildString, который по сути делает это же
```kotlin
val res = buildString {
            for(i in s)
                append(
                    if(i.isLatinLower())
                        lowers[25 - (i.code - lowersStart)]
                    else if(i.isLatinUpper())
                        uppers[25 - (i.code - uppersStart)]
                    else
                        i
                )
        }
```

Reply:
нашел метод buildString, который по сути сам вызывает StringBuilder.apply()
```suggestion
        val res = buildString {
            for(i in s)
                append(
                    if(i.isLatinLower())
                        lowers[25 - (i.code - lowersStart)]
                    else if(i.isLatinUpper())
                        uppers[25 - (i.code - uppersStart)]
                    else
                        i
                )
        }
```

Reply:
Да, молодец)